test-cases/gutenberg/darkmode.md

```diff
@@ -38,30 +38,6 @@ Expected look:
 
 --------------------------------------------------------------------------------
 
-##### TC004
-
-### Spacer Block - Check if in DarkMode all components gets proper colors
-
--   Add a `Spacer` block
--   Check if `Spacer` block in Dark Mode looks like below
-
-Expected look:  
-![SpacerDarkMode](../resources/spacer-dark-mode.png)
```

Comment:
We may not have done so to date, but I believe we could remove this (and other) now unused resource.

Reply:
Oh, good catch! I'll remove the no-longer-used resources.

Reply:
> Oh, good catch! I'll remove the no-longer-used resources.

Heads up that I'll do this in a separate PR.

Reply:
Here is [the PR](https://github.com/wordpress-mobile/test-cases/pull/153) that will remove the no-longer-used resources.